raiden/log_config.py

```diff
@@ -66,13 +84,18 @@ def configure_logging(
                     'foreign_pre_chain': processors,
                 },
             },
-            'handlers': log_handler,
+            'handlers': combined_log_handlers,
             'loggers': {
                 '': {
                     'handlers': list(log_handler.keys()),
                     'level': logger_level_config.get('', DEFAULT_LOG_LEVEL),
                     'propagate': True,
                 },
+                '': {
```

Comment:
what should be the key here? Not sure what this relates to and why `''` works in the first place.

Reply:
that's a module you configure logger for, `''` is a root logger

Reply:
Can I have multiple root loggers with different log levels?

Reply:
no, but you can set different log level for each handler

https://stackoverflow.com/a/7507842/7166699

Reply:
I did that, but know somehow the `raiden:DEBUG` thing doesn't overwrite the default log level any more. Not sure why that worked before, as the the `logger_level_config = {'': DEFAULT_LOG_LEVEL}` by default, but that gets overwritten by user preference, but the level was set by `'level': logger_level_config.get('', DEFAULT_LOG_LEVEL)`, so any `raiden:DEBUG` should have never been picked up.

Reply:
when does this happen, with tests or when running a node?

Reply:
When running a node.

I guess the levels were corretly set by `structlog.get_logger(logger_name).setLevel(level_name)` at the end, but this doesn't seem to work with my changes.